src/borg/testsuite/archiver/create_cmd.py

```diff
@@ -478,7 +505,7 @@ def test_exclude_keep_tagged(self):
         )
         self._assert_test_keep_tagged()
 
-    def test_path_normalization(self):
+    def test_path_sanitation(self):
```

Comment:
interestingly, there is also the word "sanitization", but it has a rather specific meaning, so "sanitation" is correct here.